hw/regs/axi_rt_regs.hjson

```diff
@@ -0,0 +1,294 @@
+// Copyright 2022 ETH Zurich and University of Bologna.
+// Solderpad Hardware License, Version 0.51, see LICENSE for details.
+// SPDX-License-Identifier: SHL-0.51
+// Licensed under Solderpad Hardware License, Version 0.51, see LICENSE for details.
+//
+// Thomas Benz <tbenz@student.ethz.ch>
+
+
+{
+  name: "axi_rt"
+  clock_primary: "clk_i"
+  bus_interfaces: [
+    { protocol: "reg_iface", direction: "device" }
+  ],
+  regwidth: 32
+
+  param_list: [
+    { name: "NumMst",
+      desc: "Number of masters",
+      type: "int",
+      default: "24"
+    }
+  ],
+
+  registers: [
+
+    { multireg:
+      { name:     "rt_enable"
+        desc:     "Enable RT feature on master"
+        swaccess: "rw"
+        hwaccess: "hro"
+        count:    "NumMst"
+        cname:    "rt_enable"
+        resval:   "0"
+        fields: [
+          { bits: "0:0"
```

Comment:
I kept fields on one line in the main register file config. Could adopt its formatting style here?

Reply:
Done.